sw/airborne/firmwares/rotorcraft/autopilot_arming_yaw.h

```diff
@@ -74,6 +75,32 @@ static inline void autopilot_arming_set(bool motors_on)
   }
 }
 
+#define YAW_MUST_BE_CENTERED true
```

Comment:
AP_ARMIING_YAW_MUST_BE_CENTERED or AP_ARMIING_REQ_YAW_CENTERED?

Reply:
Hmmm, you really think this makes things more clear? It's only a local define setting designed so that the arming code becomes a bit more readable (instead of using true and falses with comments each time).